### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by breaddance** [sanchit-gandhi/whisper-flash-attention#1]
Hello, I would like to ask you a question. Regarding this modification you mentioned in the whisper inference, does it involve only adding "flash-attention"? In the case of the large model, could you please provide an approximate percentage of the improvement? I would greatly appreciate your response.

**Comment by sanchit-gandhi** [comment]
Yes, that's correct. I just augmented the model to use [xFormers](https://github.com/facebookresearch/xformers) attention, which will default to flash attention if available on the hardware.

I think I only benchmarked for the medium model, for which you can see benchmark results here: https://github.com/sanchit-gandhi/whisper-flash-attention#results

You can run the benchmark for the large model by adding the checkpoint to the list here: https://github.com/sanchit-gandhi/whisper-flash-attention/blob/b1236eb2e2c7e2d8da7493dfb615761ccfb99823/benchmark_flash_attention.py#L54
